BuildingSync.xsd

```diff
@@ -4036,6 +4036,19 @@
           </xs:sequence>
         </xs:complexType>
       </xs:element>
+      <xs:element name="MeterID" type="xs:string" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>ID of the associated meter as seen by the facility manager</xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="ParentResourceUseID" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>If the ResourceUse is a submeter, this element should link to the ResourceUse which this one is a submeter of</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:attribute name="IDref" type="xs:IDREF" use="required"/>
+        </xs:complexType>
```

Comment:
@corymosiman12 please provide better documentation if this doesn't make sense

Reply:
"If this ResourceUse is intended to represent a submetered end use ('Total Lighting', 'Heating', 'Plug load', etc.), this ResourceUse should link to a parent ResourceUse that this would 'roll up to'."